features/step_definitions/service_steps.rb

```diff
@@ -60,3 +60,7 @@
 Given /^the service has been successfully tested$/ do
   @provider.default_service.proxy.update_column(:api_test_success, true)
 end
+
+Given /^the service acts as product$/ do
+  @provider.default_service.update_attribute(:act_as_product, true)
```

Comment:
Since it is a `given`, I feel it is better (and safer) to do `update_column` because it skips callbacks.